test/config.js

```diff
@@ -298,9 +298,14 @@ captureBodyTests.forEach(function (captureBodyTest) {
     t.plan(5)
 
     var errors = request.errors
+    var transactions = request.transactions
     request.errors = function (agent, list, cb) {
       request.errors = errors
-      return cb(list, agent)
+      return cb(list)
+    }
+    request.transactions = function (agent, list, cb) {
+      request.transactions = transactions
+      return cb()
```

Comment:
Out of curiosity, what is this needed for?

Reply:
Faking that the transactions have been sent to the APM Server by calling the callback

Reply:
Maybe nothing in the code base is waiting for this to be called, but better safe than sorry - especially if this expectation changes at a later date

Reply:
That's fair. 👍